ZelBack/src/services/appsService.js

```diff
@@ -10221,6 +10226,10 @@ async function checkInstallingAppPortAvailable(portsToTest = []) {
     await serviceHelper.delay(10 * 1000);
     // eslint-disable-next-line no-await-in-loop
     let askingIP = await fluxNetworkHelper.getRandomConnection();
+    while (askingIP.split(':')[0] === myIP) {
```

Comment:
it's a while because it's on the method that is executed to check if the app can be installed on that node, I don't think it's ok to answer false because the random node that was selected it's under the same ip. And there is no cache on getRandomConnection so, what is the probability of that runs more than 2 times? idk.

Reply:
oh yes you are correct